### ISSUE TITLE
1.17

### ISSUE THREAD

**Issue by UltraBlackLinux** [liachmodded/brandpacket#4]
1.17...

please.

**Comment by liach** [comment]
Hmm, does the current build crashes with 1.17? Mind post a crash report if it does.

Unfortunately, I do not turn on notifications here, so sorry for the late reply.

**Comment by UltraBlackLinux** [comment]
> Hmm, does the current build crashes with 1.17? Mind post a crash report if it does.
> 
> Unfortunately, I do not turn on notifications here, so sorry for the late reply.

yeah, I think it did.
You need to update the mod, as a LOT changed in 1.17.
Here are the resources you need:
https://fabricmc.net/wiki/tutorial:migratemappings
https://github.com/FabricMC/fabric-example-mod/commit/54035654b75dd756d98f89fde179cb49183a6d68#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7
https://github.com/FabricMC/fabric-example-mod/compare/29c522536fc16233833221e22eed3f106c0726bc...1.17
https://fabricmc.net/versions.html
You just have to try around for a bit, it took me a long time to figure out, how to do it the first time I tried

**Comment by liach** [comment]
yeah i know how things work out; am just too lazy and a bit busy recently. this is actually a one-line patch on mostly stable code, so i expected to be fine.
